### PR TITLE
Ruff and project standardization

### PR DESCRIPTION
**Pull request by bandophahita**
implementing ruff and project standardization
https://github.com/ScreenPyHQ/screenpy/issues/127

### CODE REVIEW

File: tests/test_adapter.py [hunk 1]
```diff
@@ -88,7 +102,11 @@ def test_aside(self, mocked_allure, narrator_level, allure_level):
         mocked_allure.step.assert_called_once_with(aside_message)
         mocked_allure.severity.assert_called_once_with(allure_level)
 
-    def test_error(self, mocked_manager, mock_allure_trappings):
+    def test_error(
+        self,
+        mocked_manager: mock.Mock,  # noqa: ARG002
```
Comment (perrygoy) [inline tests/test_adapter.py]: I think this `mocked_manager` was actually an accidental copy-paste error, or maybe i forgot to delete it after changing strategies. You can (probably) remove it!

I don't know why `pytest` hasn't been complaining about an unknown fixture though.

**Review by perrygoy — COMMENTED** [review]
Only one comment which we might want to address!

File: tests/test_adapter.py (hunk 1 above)
Comment (bandophahita) [reply tests/test_adapter.py]: I didn't dig into it, but removing it causes the test to fail and I don't know why.  I figured there was some magic going on.

Comment (bandophahita) [reply tests/test_adapter.py]: ahh. found it.  It shouldn't be called mocked_manager. It's technically the mocked_allure.  The patch is happening at the class level. I updated it to make more sense.

File: screenpy_adapter_allure/__version__.py [hunk 2]
```diff
@@ -1,16 +1,12 @@
 """
 ScreenPy Adapter: Allure
 """
+
 # (the ascii art would be too long for this one :( )
```
Comment (perrygoy) [inline screenpy_adapter_allure/__version__.py:5]: Still sad about this.

**Review by perrygoy — APPROVED** [review]
Very very nice, thanks @bandophahita!

**Pull request merged**